### PR TITLE
fix: prevent duplicate multiple choice

### PR DESCRIPTION
**Pull request by thywang**
## Notion ticket link

[Prevent same option from being created for multiple choice](https://www.notion.so/uwblueprintexecs/Prevent-same-option-from-being-created-for-multiple-choice-51dc19f83b7f426ab007360a0a5b12ff)

## Implementation description
* Implemented error handling when options for multiple choice or multi-select are not unique
* Refactored error handling for options to use enum 

## Steps to test
1. Navigate to `<baseURL>/admin/add-camp`
2. On the Registration Form page, add a new question
3. Choose multiple choice as Question Type and try Save -> should see error You must enter at least 2 options
4. Choose checkbox as Question Type and try Save -> should see error You must enter at least 1 option
5. Choose multiple choice as Question Type, add a second option, type some value into one of the options and try Save -> should see error Options cannot be empty
6. Enter the same value into the other option and try Save -> should see error Duplicate options are not allowed
7. Change one option's value and Save -> should be successful
8. Edit the question and choose check box as Question Type
9. Type something as option value and Save -> should be successful

## What should reviewers focus on?
* Ensure behaviour of the form to create a question works as expected

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx [hunk 1]
```diff
@@ -39,6 +39,14 @@ type AddQuestionModalProps = {
   isFormTemplatePage?: boolean;
 };
 
+enum OptionErrorStatus {
+  NO_OPTION = "Options cannot be empty",
+  AT_LEAST_ONE = "You must enter at least 1 option",
+  AT_LEAST_TWO = "You must enter at least 2 options",
+  NO_DUPLICATES = "Duplicate options are not allowed",
```
Comment (williamtran10) [inline frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx:46]: Can you change `NO_OPTION` to `EMPTY_OPTION` and `NO_DUPLICATES` to `HAS_DUPLICATES`? I think those are better descriptors.

**Review by williamtran10 — CHANGES_REQUESTED** [review]
Great job Tina! Overall looks good and it fixes the bug so that's good. I requested that you change 2 of the names though. Also, I see some lint warnings in your code: it would be great if you can fix those too (hint: look in the `README`)

File: frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx (hunk 1 above)
Comment (thywang) [reply frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx]: Thanks William! I agree they are better descriptors. I have changed the names and fixed the lint warnings in my latest commit.

**Review by BraydenRoyston — APPROVED** [review]
Tested on my end and everything seems to be working well. Approved by me pending William's suggestions which I agree with. 

Great job team! 🎉

**Review by williamtran10 — APPROVED** [review]

**Pull request merged**